tests/frontend/reasoning_effort/test_reasoning_effort.py

```diff
@@ -0,0 +1,201 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""End-to-end tests covering reasoning effort behaviour."""
+
+from __future__ import annotations
+
+import logging
+import os
+import shutil
+from typing import Any, Dict, Optional, Tuple
+
+import pytest
+import requests
+
+from tests.fault_tolerance.test_request_migration import DynamoFrontendProcess
```

Comment:
Maybe copy the DynamoFrontendProcess class here?

In case something is changed at the migration tests, i.e. frontend port, it would be hard to understand why it is breaking this test here.